Assets/_MUTUAL/Screens/LoadCaseScreen.cs

```diff
@@ -0,0 +1,165 @@
+        private Transform m_Parent;
+        private List<GameObject> m_Meshes;
+
+        #endregion
+
+        #region Constructors
+
+        public LoadCaseScreen(Project project)
+        {
+            m_Project = project;
+            m_Parent = GameObject.FindGameObjectWithTag(ParentTag).transform;
+            m_Meshes = new List<GameObject>();
+        }
+
+        #endregion
+
+        #region Public Methods
+
+        public void ActivateScreen()
+        {
+            Debug.Log("Activated");
+            PopulateUiElements();
+            AttachListeners();
+        }
+
+        public void DeactivateScreen()
+        {
+            Camera.main.gameObject.SetActive(true);
+            DettachListeners();
+            UnPopulateUiElements();
+        }
+
+        public void OnLoadCaseButtonClicked()
+        {
+            Debug.Log("Load");
+            string path = EditorUtility.OpenFilePanel("Select case file", "", "CASE");
+            if (path.Length != 0)
+            {
+                m_Project.LoadProject(path);
+            }
+
+            Debug.Log($"Patient Details: {m_Project.PatientData.ToString()}");
+
+            var meshKeys = string.Empty;
+            foreach (var key in m_Project.MeshGeoms.Keys)
+            {
+                meshKeys += (key + ", ");
+            }
+            Debug.Log($"Meshes: {meshKeys}");
+
+            var landmarks = string.Empty;
+            foreach (var lm in m_Project.LandMarks)
+            {
+                landmarks += $"Bone:{lm.Bone}, Type: {lm.Type}, Position: {lm.Position}\n";
+            }
+            Debug.Log($"Landmarks: \n{landmarks}");
+
+            var cases = string.Empty;
+            foreach (var plan in m_Project.PlanValues)
+            {
+                foreach (var measurement in plan)
+                {
+                    cases += $"{measurement.Key} {measurement.Value}\n";
+                }
+                cases += $"\n";
+            }
+            Debug.Log($"Cases: \n{cases}");
+
+            var functionalValues = string.Empty;
+            foreach (var value in m_Project.FunctionalValues)
+            {
+                functionalValues += $"ID: {value.IdFunctionalValues}, " +
+                    $"Measure: {value.Measure}, Value: {value.MeasureValue}\n";
+            }
+            Debug.Log($"Functional values: \n{functionalValues}");
+        }
+
+        public void OnViewButtonClicked()
+        {
+            Debug.Log("View");
+
+            foreach (var go in m_Meshes)
+            {
+                go.SetActive(false);
+                UnityEngine.Object.DestroyImmediate(go);
+            }
+            m_Meshes.Clear();
+
+
+            var material = Resources.Load<Material>("BoneMaterial");
+            foreach (var mesh in m_Project.MeshGeoms)
+            {
+                var go = new GameObject(mesh.Key, typeof(MeshFilter), typeof(MeshRenderer));
+                go.transform.parent = m_Parent;
```

Comment:
Meshes should have a different parent to this (which is currently a 2D canvas UI).

Reply:
@KICO-SOURCE Separate parent defined.